worker/mozfest_session_proposal_2016.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function(notifier_messager, mailroom) {
+  var LUMBERYARD_EVENT = "mailer";
+  var FROM_EMAIL = "MozFest <festival@mozilla.org>";
+  var SAWMILL_EVENT = "mozfest_session_proposal_2016"
+
+  return function(event, cb) {
+    if (event.event_type !== SAWMILL_EVENT) {
+      return process.nextTick(cb);
+    }
+    var mail = mailroom.render(SAWMILL_EVENT, {
+      first_name: event.data.first_name,
+      github_issue_url: event.data.github_issue_url,
+      github_issue_title: event.data.github_issue_title
```

Comment:
@mmmavis this variable and the three above it (as well as the `SAWMILL_EVENT` variable) should give you all the info you need to set up the hook for sending emails on the mozfest server